crates/ruff_python_formatter/tests/snapshots/format@stub_files__suite.pyi.snap

```diff
@@ -309,4 +309,27 @@ class ComplexStatements:
 ```
 
 
+## Preview changes
+```diff
+--- Stable
++++ Preview
+@@ -110,6 +110,7 @@
+ 
+     class InnerClass5:
+         def a(self): ...
++
+     field1 = 1
+ 
```

Comment:
This seems like a bug in Black's implementation: https://github.com/psf/black/issues/4113